### PR TITLE
Suppress excess logging, add error output

### PR DESCRIPTION
**Pull request by derrickburns**
@tjotala This is an example of a change we would want to push out quickly.  We need a process that enables that.  Right now, I am under the impression that I have to create a Jira ticket just to get this approved.

### CODE REVIEW

**Comment by derrickburns** [comment]
@HaroldBernard Perhaps we can create a "fasttrack" jira ticket using your tidebot? 

`/jira` -> would create a ticket with the same title as the PR, assign it to the person making the comment, move it to 'In progress' state with a back reference to the GH PR, and send a Jira message to @tapani requesting approval?

**Comment by tjotala** [comment]
> @tjotala This is an example of a change we would want to push out quickly. We need a process that enables that. Right now, I am under the impression that I have to create a Jira ticket just to get this approved.

Our SOP for development (https://docs.google.com/document/d/19vWtyDNesVyxE5gKXwppgRGWvn-EXrttCLJH5nRys1U/edit#) has allowances for _trivial_ changes with respect to reviews. It also notes that GitHub is part of our "Design History File" and for a trivial change such as this I think PR approval in GitHub will suffice.

**Comment by tjotala** [comment]
> Any reason that it is not building?
> 
> Changes look ok - although not completely sure what it means to take out the accessLogMiddleware

@derrickburns Looks to me like you're removing the access log from middleware stack & I'm assuming that is redundant with the API gateway in place doing the same. Can you confirm? Assuming we would not deploy any services in the legacy environment (sans the API gateway), that seems OK.

**Comment by derrickburns** [comment]
Correct

File: service/api/api.go [hunk 1]
```diff
@@ -55,10 +55,10 @@ func (a *API) InitializeMiddleware() error {
 	if err != nil {
 		return err
 	}
-	accessLogMiddleware, err := middleware.NewAccessLog()
-	if err != nil {
-		return err
-	}
+	//accessLogMiddleware, err := middleware.NewAccessLog()
```
Comment (darinkrauss) [inline service/api/api.go]: I would completely remove (not just comment out). The code is in the git history so can be recovered if ever necessary.

File: task/service/service/service.go [hunk 2]
```diff
@@ -200,9 +200,9 @@ func (s *Service) initializeDexcomClient() error {
 
 	dxcmPrvdr, err := dexcomProvider.New(s.ConfigReporter().WithScopes("provider"), s.dataSourceClient, s.TaskClient())
 	if err != nil {
-		s.Logger().Warn("Unable to create dexcom provider")
+		s.Logger().WithError(err).Warn("Unable to create dexcom provider")
 	} else {
-		s.Logger().Debug("Loading dexcom client config")
+		s.Logger().WithError(err).Debug("Loading dexcom client config")
```
Comment (darinkrauss) [inline task/service/service/service.go]: No need for `.WithError(err)` here since, by definition, `err` is `nil` according to the logic here.

Comment (darinkrauss) [reply task/service/service/service.go]: BTW, no harm to include this here, but it just won't ever do anything.

**Pull request merged**